src/map/clif_packetdb.hpp

```diff
@@ -2041,6 +2041,10 @@
 	parseable_packet( HEADER_CZ_REQ_EMOTION_EXPANSION, sizeof( struct PACKET_CZ_REQ_EMOTION_EXPANSION ), clif_parse_dull, 0 );
 #endif
 
+#if PACKETVER_MAIN_NUM >= 20230802
+	parseable_packet( HEADER_CZ_QUEST_STATUS_REQ, -1, clif_parse_dull, 0 );
```

Comment:
So length of the packet is dynamic?

Reply:
Yes, the size in the client is defined with minimum 4 and variable maximum.